src/pages/CategoryPage.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const CategoryPage = () => <button type="button">Check status</button>;
+
+export default CategoryPage;
```

Comment:
Dear @amklass11  and @TadesseDev, In my opinion, these requirements are for the next milestones as the actions are not being dispatched yet.

Happy coding!

Reply:
=Thank you for the information @GedeonTS  but I dont think so, I try to look through the requirements and only editing tasks and one more requirement (I forget the name) were not part of this requirement, But everything else was part of this milestone.

Reply:
We are all right @TadesseDev, seems like the author submitted the wrong for your review because on mine it was `Bookstore: add reducers and actions`, so everything is fine now

Reply:
That makes sense @GedeonTS. Good day 🙌